bin/commands/runs.js

```diff
@@ -41,6 +41,12 @@ module.exports = function run(args) {
           return build.createBuild(bsConfig, zip).then(function (data) {
             let message = `${data.message}! ${Constants.userMessages.BUILD_CREATED} with build id: ${data.build_id}`;
             let dashboardLink = `${Constants.userMessages.VISIT_DASHBOARD} ${config.dashboardUrl}${data.build_id}`;
+            if ((utils.isUndefined(bsConfig.run_settings.parallels) && utils.isUndefined(args.parallels)) || (!utils.isUndefined(bsConfig.run_settings.parallels) && bsConfig.run_settings.parallels == Constants.constants.DEFAULT_PARALLEL_MESSAGE)) {
```

Comment:
Build gets submitted successfully in case of DEFAULT_PARALLEL_MESSAGE. 
https://github.com/browserstack/browserstack-cypress-cli/pull/45/files#diff-0f95deee3b5633fae4d46a02533b269cR72